### ISSUE TITLE
`exclude-file-type=yml` doesnt ignore `yaml` files and vice-versa.

### ISSUE THREAD

**Issue by gurukiran07** [Boeing/config-file-validator#214]
`validator --exclude-file-types=yml` doesnt exclude `.yaml`. It only excludes `yml` files. 

Expected output: To be able to exclude files both `yaml` and `yml` when given one of them as argument to `exclude-file-type`.

**Comment by kehoecj** [comment]
Yeah we should definitely fix this. I agree with @ccoVeille on this assessment:

>Maybe we should add an field on the filetype to list the known named, or the additional and optional aliases for a file type

**Comment by bhusal-rj** [comment]
I would like to work on this issue. Can you please assign me,

**Comment by kehoecj** [comment]
> I would like to work on this issue. Can you please assign me,

@bhusal-rj done!

**Comment by bhusal-rj** [comment]
![Image](https://github.com/user-attachments/assets/4f804075-a8bc-4f8b-a7a3-aa778f815d3e)

Looks like the bug is already fixed

**Comment by kehoecj** [comment]
@bhusal-rj Sorry, it was already fixed in a previous PR. Thank you for taking a look and confirming!
